string cell_name
---
float64 x
float64 y
